cmd/evm/eofparser.go

```diff
@@ -161,3 +161,24 @@ func parseAndValidate(s string) (*vm.Container, error) {
 	}
 	return &c, nil
 }
+
+func eofDump(ctx *cli.Context) error {
```

Comment:
we have `cmd/rlpdump` and `cmd/abidump`. How about a `cmd/eofdump` ?

Reply:
Yep, will move

Reply:
ah @fjl actually said today that he would prefer to have it a subcommand of evm